modules/core/src/main/java/org/apache/ignite/internal/IgniteKernal.java

```diff
@@ -293,52 +294,79 @@
+ * </ul>
+ * The <tt>start()</tt> method also perfoms additional validation of the provided {@link IgniteConfiguration} and
+ * prints some suggestions such as:
+ * <ul>
+ * <li>Ignites configuration optimizations (e.g. disabling {@link EventType} events).</li>
+ * <li>{@link JvmConfigurationSuggestions} optimizations</li>
+ * <li>{@link OsConfigurationSuggestions} optimizations</li>
+ * </ul>
  */
 public class IgniteKernal implements IgniteEx, IgniteMXBean, Externalizable {
-    /** */
+    /** Class serialization version number. */
     private static final long serialVersionUID = 0L;
 
-    /** Ignite site that is shown in log messages. */
+    /** Ignite web-site that is shown in log messages. */
     public static final String SITE = "ignite.apache.org";
 
     /** System line separator. */
     private static final String NL = U.nl();
 
-    /** Periodic starvation check interval. */
+    /**
+     * Default interval of checking thread pool state for the starvation. Will be used only if the
+     * {@link IgniteSystemProperties#IGNITE_STARVATION_CHECK_INTERVAL} system property is not set.
+     * <p>
+     * Value is {@code 30 sec}.
+     */
     private static final long PERIODIC_STARVATION_CHECK_FREQ = 1000 * 30;
 
-    /** Force complete reconnect future. */
+    /** Object is used to force completion the previous reconnection attempt. See {@link ReconnectState} for details. */
     private static final Object STOP_RECONNECT = new Object();
 
-    /** Separator for formatted coordinator properties. */
+    /** The separator is used for coordinator properties formatted as a string. */
     public static final String COORDINATOR_PROPERTIES_SEPARATOR = ",";
 
-    /** Default long operations dump timeout. */
+    /**
+     * Default timeout in milliseconds for dumping long running operations. Will be used if the
+     * {@link IgniteSystemProperties#IGNITE_LONG_OPERATIONS_DUMP_TIMEOUT} is not set.
+     * <p>
+     * Value is {@code 60 sec}.
+     */
     public static final long DFLT_LONG_OPERATIONS_DUMP_TIMEOUT = 60_000L;
 
-    /** Long jvm pause detector. */
+    /** Currently used instance of JVM pause detector thread. See {@link LongJVMPauseDetector} for details. */
     private LongJVMPauseDetector longJVMPauseDetector;
 
-    /** */
+    /** The main kernal context which holds all the {@link GridComponent}s. */
     @GridToStringExclude
     private GridKernalContextImpl ctx;
 
-    /** Helper that registers MBeans */
+    /** Helper which is register and unregister MBeans. */
     @GridToStringExclude
     private IgniteMBeansManager mBeansMgr;
 
-    /** Configuration. */
+    /** Ignite runtime configuration instance. */
     private IgniteConfiguration cfg;
 
-    /** */
+    /** Ignite logger instance which enriches log messages with the node instance name and the node id. */
     @GridToStringExclude
     private GridLoggerProxy log;
 
-    /** */
+    /** Instance name of Ignite node. */
```

Comment:
`Name of Ignite node`? Does not _instance_ and _node_ have the same meaning here?

Reply:
Agree, fixed.